ddtrace/contrib/anthropic/utils.py

```diff
@@ -0,0 +1,284 @@
+import sys
+from typing import AsyncGenerator
+from typing import Dict
+from typing import Generator
+from typing import Tuple
+
+from ddtrace.internal.logger import get_logger
+from ddtrace.internal.utils import get_argument_value
+from ddtrace.vendor import wrapt
+
+
+log = get_logger(__name__)
+
+
+def handle_stream_response(integration, resp, args, kwargs, span):
+    if _is_async_generator(resp):
+        return TracedOpenAIAsyncStream(resp, integration, span, args, kwargs)
+    elif _is_generator(resp):
+        return TracedOpenAIStream(resp, integration, span, args, kwargs)
+    elif _is_stream_manager(resp):
+        return TracedOpenAIStreamManager(resp, integration, span, args, kwargs)
+
+
+def _process_finished_stream(integration, span, args, kwargs, streamed_chunks, message_accumulated=False):
+    messages = None
+    messages = kwargs.get("messages", None)
+    chat_messages = get_argument_value(args, kwargs, 0, "messages")
```

Comment:
Sorry - just realized the openai integration has terrible variable naming. Can we change the names to `input_messages` and `output_messages` to be more clear what we're passing?

Reply:
Honestly it would probably make this a bit cleaner to just pass in `kwargs` to `llmobs_set_tags()` similar to how we do it in the openai integration, since we're not actually doing anything with the input messages in this function.